### PR TITLE
Added Grafana questions and answers

### PR DESCRIPTION
**Pull request by AbyelT**
Adding answers to questions in https://github.com/bregman-arie/devops-exercises/issues/4. Also added new questions and answers

### CODE REVIEW

**Comment by AbyelT** [comment]
Hello @bregman-arie, i added some answers to existing questions from https://github.com/bregman-arie/devops-exercises/issues/4 and created some new questions about using and organizing an Grafana instance. I'm not fully done yet (some typo checking, references) but would love to hear feedback if you have any. Thanks in advance :)

File: exercises/grafana/README.md [hunk 1]
```diff
@@ -5,3 +5,154 @@
...
+</b></details>
+
+<details>
+<summary>Explain how we can enforce HTTPS</summary><br><b>
+
+[Grafana community](https://grafana.com/docs/grafana/latest/getting-started/getting-started/): Set the protocol to https on the Configuration settings, Grafana will then expect clients to send requests using the HTTPS protocol. Any client that uses HTTP will receive an SSL/TLS error.
+</b></details>
+
+<details>
+<summary>How can we install plugins</summary><br><b>
```
Comment (bregman-arie) [inline exercises/grafana/README.md:45]: How can we install plugins for Grafana?*

File: exercises/grafana/README.md [hunk 2]
```diff
@@ -5,3 +5,154 @@
...
+
+<details>
+<summary>What is the format for the dashboard</summary><br><b>
+
+[Grafana docs](https://grafana.com/docs/grafana/latest/dashboards/json-model/): Grafana dashboards are represented in JSON files as objects, they store metadata about a dashboard e.g. dashboard properties, panel metadata and variables.
+
+</b></details>
+
+<details>
+<summary>How is the version upgrade done</summary><br><b>
```
Comment (bregman-arie) [inline exercises/grafana/README.md:113]: Same as "on which OS Grafana is supported?"

**Review by bregman-arie — CHANGES_REQUESTED** [review]
Thanks for the effort. Left some comments.

**Comment by AbyelT** [comment]
Thanks for the feedback! I agree that some questions are redundant and the user can check the docs instead. I think the Q&A for Grafana is ready to merge, feel free to let me know if there is something to be addressed. Thanks in advance!

**Review by bregman-arie — APPROVED** [review]

**Comment by bregman-arie** [comment]
Thanks for the effort

**Pull request merged**